remotespark/magics/sparkmagicsbase.py

```diff
@@ -47,7 +47,7 @@ def _sqlquery(cell, samplemethod, maxrows, samplefraction):
 
     @staticmethod
     def print_endpoint_info(info_sessions):
-        sessions_info = ["        {}".format(i) for i in info_sessions]
+        sessions_info = ["        {}\n".format(i) for i in info_sessions]
```

Comment:
I agree but I want to get this out first and do it in another commit. Will create an GH issue for it.